vnext/ReactUWP/Modules/Animated/NativeAnimatedNodeManager.cpp

```diff
@@ -230,16 +230,19 @@ namespace react {
       if (m_eventDrivers.count(key))
       {
         auto& drivers = m_eventDrivers.at(key);
+        auto iterator = drivers.begin();
 
-        for (auto iterator = drivers.begin(); iterator != drivers.end(); iterator++)
```

Comment:
I would prefer to create a erase_if template for this kind of problem. [https://en.cppreference.com/w/cpp/experimental/map/erase_if](https://en.cppreference.com/w/cpp/experimental/map/erase_if).
Also it's better to still use for loop like this:
for (auto i = c.begin(), last = c.end(); i != last; ) {
  if (pred(*i)) {
    i = c.erase(i);
  } else {
    ++i;
  }
}